devmand/gateway/src/devmand/channels/cli/IoConfigurationBuilder.cpp

```diff
@@ -46,18 +46,14 @@ IoConfigurationBuilder::IoConfigurationBuilder(
       deviceConfig.ip,
       plaintextCliKv.at("username"),
       plaintextCliKv.at("password"),
-      plaintextCliKv.find("flavour") != plaintextCliKv.end()
-          ? plaintextCliKv.at("flavour")
-          : "",
+      loadConfigValue(plaintextCliKv, "flavour", ""),
       std::stoi(plaintextCliKv.at("port")),
-      loadTimeout(
-          plaintextCliKv,
-          configKeepAliveIntervalSeconds,
-          defaultKeepaliveInterval),
-      loadTimeout(
-          plaintextCliKv,
-          configMaxCommandTimeoutSeconds,
-          defaultCommandTimeout));
+      toSeconds(loadConfigValue(
+          plaintextCliKv, configKeepAliveIntervalSeconds, "60")),
+      toSeconds(
+          loadConfigValue(plaintextCliKv, configMaxCommandTimeoutSeconds, "5")),
```

Comment:
Default should be around 60 seconds
for ssh connection timeout 30 seconds

Reply:
intervals updated, added quiet period